### PR TITLE
API docs: Add `basics` and `operations` chapters.

### PR DESCRIPTION
**Pull request by lukasgraf**
Adds two more chapters to the API docs:

- **Basics** - describes how the overall interaction with the API happens
- **Operations** - Describes what HTTP Methods are mapped to which operation

@deiferni

### CODE REVIEW

File: docs/api/basics.rst [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+weitergeleitet wird.
+
+**Beispiel-Request**:
+
+.. sourcecode:: http
+
+  GET /ordnungssystem HTTP/1.1
+  Accept: application/json
+
+Dieser Header muss bei *jedem* Request auf die API gesetzt werde, und wird in
```
Comment (deiferni) [inline docs/api/basics.rst]: werde -> werden

File: docs/api/basics.rst [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+den folgenden Beispielen daher nicht mehr immer explizit erwähnt werden.
```
Comment (deiferni) [inline docs/api/basics.rst]: ~~werden~~

**Comment by deiferni** [comment]
:+1:

**Comment by lukasgraf** [comment]
@deiferni thanks, updated

**Pull request merged**